packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -843,61 +883,158 @@ class SemanticsProperties extends DiagnosticableTree {
   /// If a label is provided, there must either by an ambient [Directionality]
   /// or an explicit [textDirection] should be provided.
   ///
+  /// Only one of the [label] or [attributedLabel] should be provided.
```

Comment:
should -> must

(here and elsewhere)

Reply:
It is also ok if none of them is provided. I thought the `must` imply at least one of them need to be provide

Reply:
> Callers must not provide both [label] and [attributedLabel]. One or both must be null.